doc.go

```diff
@@ -505,6 +505,11 @@ To address this issue and prevent potential downstream disruptions, the context
 When enabled, this feature iterates through all values in string columns, identifying and replacing any invalid characters with `�`.
 This ensures that Arrow records conform to the UTF-8 standards, preventing validation failures in downstream services like the Rust Arrow library that impose strict validation checks.
 
+### WithHigherPrecision in Arrow batches
+To preserve BigDecimal values within Arrow batches, set the `enableHigherPrecision` value to true in the context using `WithHigherPrecision`.
```

Comment:
I'm not sure we should mention internal parameter name. Maybe just `use WithHigherPrecision` is good enough?

Reply:
done!